README.md

```diff
@@ -1 +1,583 @@
-# microservices-observability
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/microservices-observability.svg?branch=master)](https://travis-ci.org/ballerina-guides/microservices-observability)
+
+# Observing Ballerina Microservices
+
+Observability of services is useful to retrieve internal information for easier troubleshooting and system status analysis. Ballerina allows to observe its services using Monitoring, Logging and Distributed tracing methods, with the use of other external tools.
+
+Ballerina observability can be categorized into two types as,
+- Default observability - Each Ballerina service can expose its internal information just by enabling the built-in observability. (Eg : Network call duration)
+- User-defined observability - More specific data can be retrieved by using user-defined observability. (Eg : Function execution time) This requires the explicit import of the package `ballerina/observe`.
+
+> This guide is a modification of the [Parallel Service Orchestration guide](https://github.com/ballerina-guides/parallel-service-orchestration) to explain a real-world use case of Ballerina observability.
+
+Following are the sections available in this guide.
+
+- [Introduction](#Introduction)
+- [Prerequisites](#prerequisites)
+- [Testing the services](#Testing_the_services)
+- [Observability](#observability)
+- [Deployment](#deployment)
+
+## Introduction
+
+This guide explains the applicability of Ballerina observability in below scenarios. 
+- Identifying the bottleneck
+- Identifying the unavailble services
+- Determining database call duration and bottleneck queries
+- Function execution time (user-defined observability)
+
+Consider a real-world use case of a travel agency that arranges complete tours for users. A tour package includes airline ticket reservation, hotel room reservation and car rental. Therefore, the travel agency service requires communicating with other necessary back-ends. Those back-ends query their databases to retrieve information of availability of their services.
+
+Travel Agency service communicates with multiple services. Airline reservation service has multiple resources for three different airlines, whereas Hotel reservation and Car rental services only have a single resource. The travel agency service checks all airline resources in parallel to select the cheapest available. Each associated service then queries their database for the availability and information of their services. Only Hotel resevation service does not have a associated database. Rather the response is hard-coded to demonstrate the difference in traces.
```

Comment:
Hotel reservation -> hotel reservation
Car rental -> car rental

Shall we rephrase the last two sentences?